src/primordials.ts

```diff
@@ -0,0 +1,4 @@
+export const primordials: any = // TODO export as typeof import('node-primordials'), types are wrong
+  // @ts-expect-error
+  typeof primordials !== 'undefined' ? primordials /* c8 ignore next */ :
```

Comment:
I don't think `c8 ignore next` is a thing, is that new? I thought it only supported `/* c8 ignore start */` and `/* c8 ignore stop */`

Reply:
it is. you can also specify the amount of lines. https://github.com/bcoe/c8/blob/main/README.md#ignoring-uncovered-lines-functions-and-blocks

Reply:
Oh, that's somewhat new-ish, I think. (Like, added in the last 5 years or so probably lol.) Nice!